Update WSL kernel version

crahrig opened the pull request:
Updating the minimum WSL kernel version needed to work with Windows 10, version 21H2.

opbld31 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/chaining?branch=pr-en-us-43) |
[docs/chaining.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/chaining.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/chaining?branch=pr-en-us-43) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C11%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810112205484574-43%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=WzgPIBUkkyLGVjNzTEI5c8eRwjzZW551ytDgACZYW%2Bc%3D&st=2018-10-11T22%3A02%3A10Z&se=2018-11-11T22%3A07%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

crahrig committed:
Update WSL kernel version

Updating the minimum WSL kernel version needed to work with Windows 10, version 21H2.

crahrig committed:
Update WSL kernel version

Updating the minimum WSL kernel version needed to work with Windows 10, version 21H2.
